packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -742,6 +742,27 @@ class ClipRect extends SingleChildRenderObjectWidget {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=eI43jkQkrvs}
 ///
+/// {@tool dartpad}
+/// This example shows various [ClipRRect] applied to containers.
+///
+/// ** See code in examples/api/lib/widgets/basic/clip_rrect.0.dart **
+/// {@end-tool}
+///
+/// ## Troubleshooting
+///
+/// ### Why my ClipRRect child doesn't have rounded corners?
+///
+/// When [ClipRRect] is bigger than the child it contains, its rounded corners
+/// could be drawn not where expected. Make sure that [ClipRRect] and its child
```

Comment:
```suggestion
/// could be drawn in unexpected positions. Make sure that [ClipRRect] and its child
```
Maybe describe in what way the location is surprising.